src/NumericalAlgorithms/Spectral/Spectral.hpp

```diff
@@ -49,6 +49,18 @@ namespace Spectral {
  *
  * \details Choose `Legendre` for a general-purpose DG mesh, unless you have a
  * particular reason for choosing another basis.
+ *
+ * \warning The `FiniteDifference` "basis" is used to denote that only the
+ * collocation points are defined, but that differentiation, integration, and
+ * interpolation schemes are to be chosen locally wherever a `FiniteDifference`
+ * mesh is being used. The reason is that there isn't a requirement that the
+ * basis and collocation point locations are at all related to the
+ * differentiation, integration, or interpolation methods, it is merely a
```

Comment:
[nitpick] I would replace the last comma with a dash (for lack of em-dashes in ascii)
"methods, it is" -> "methods — it is"